<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vAgregarPasajero</class>
 <widget class="QDialog" name="vAgregarPasajero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Personas</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>191</width>
     <height>270</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtfNombre"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Apellidos:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtfID"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tel:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtfTel"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Nacionalidad:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtfNacionalidad"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Destino:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cmbxDestino">
      <item>
       <property name="text">
        <string>Guanacaste</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cartago</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Heredia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Puntarenas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pe√±as Blancas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Managua</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Golfito</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Panama</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Tickets:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spnbxTickets"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Estatura:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Peso:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtfApellidos"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="txtfEstatura"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="txtfPeso"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>84</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Pasajero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAgregarPasajero">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Pasajero</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
